.github/workflows/integration.yml

```diff
@@ -70,3 +70,9 @@ jobs:
     uses: tarantool/tarantool-c/.github/workflows/reusable_testing.yml@master
     with:
       artifact_name: tarantool-ubuntu-focal-${{ github.sha }}
+
+  php-queue:
+    needs: tarantool
+    uses: ylobankov/queue/.github/workflows/reusable_qa.yml@add-reusable-testing-workflow
```

Comment:
Change

`ylobankov/queue` -> `tarantool-php/queue`
`add-reusable-testing-workflow` -> `master`

after https://github.com/tarantool-php/queue/pull/18 is merged.

Reply:
Done.